doctypes/account-link.md

```diff
@@ -54,31 +54,31 @@ When resolving an account-link that is linked to a 3ID the result looks like the
 ### Ethereum account-link
 ```JSON
 {
-  "owners": ["0xab16a96d359ec26a11e2c2b3d8f8b8942d5bfcdb"],
+  "owners": ["0xab16a96d359ec26a11e2c2b3d8f8b8942d5bfcdb@eip155:1"],
   "content": "did:3:bafyreiecedg6ipyvwdwycdjiakhj5hiuuutxlvywtkvckwvsnu6pjbwxae"
 }
 ```
 
 ### Polkadot account-link
 ```JSON
 {
-  "owners": ["5hmuyxw9xdgbpptgypokw4thfyoe3ryenebr381z9iaegmfy"],
+  "owners": ["5hmuyxw9xdgbpptgypokw4thfyoe3ryenebr381z9iaegmfy@polkadot:polkadot-chain"],
   "content": "did:3:bafyreiecedg6ipyvwdwycdjiakhj5hiuuutxlvywtkvckwvsnu6pjbwxae"
 }
 ```
 
 ### Cosmos account-link
 ```JSON
 {
-  "owners": ["cosmos1t2uflqwqe0fsj0shcfkrvpukewcw40yjj6hdc0"],
+  "owners": ["cosmos1t2uflqwqe0fsj0shcfkrvpukewcw40yjj6hdc0@cosmos:cosmoshub-3"],
   "content": "did:3:bafyreiecedg6ipyvwdwycdjiakhj5hiuuutxlvywtkvckwvsnu6pjbwxae"
 }
 ```
 
 ### Bitcoin account-link
 ```JSON
 {
-  "owners": ["128Lkh3S7CkDTBZ8W7BbpsN3YYizJMp8p6"],
+  "owners": ["128Lkh3S7CkDTBZ8W7BbpsN3YYizJMp8p6@bip122:000000000019d6689c085ae165831e93"],
```

Comment:
Cool so this could be used to link to any bitcoin fork?

Reply:
Yes, literally any BIP122 compatible similar to how Ethereum forks that are compatible with EIP155.

Examples from [CAIP-2](https://github.com/ChainAgnostic/CAIPs/blob/master/CAIPs/caip-2.md)
```
# Bitcoin mainnet (see https://github.com/bitcoin/bips/blob/master/bip-0122.mediawiki#definition-of-chain-id)
bip122:000000000019d6689c085ae165831e93

# Litecoin
bip122:12a765e31ffd4059bada1e25190f6e98

# Feathercoin (Litecoin fork)
bip122:fdbe99b90c90bae7505796461471d89a
```